docs/interpreter/jdbc.md

```diff
@@ -167,6 +167,10 @@ There are more JDBC interpreter properties you can specify like below.
     <td>default.jceks.credentialKey</td>
     <td>jceks credential key</td>
   </tr>
+  <tr>
+    <td>zeppelin.interpreter.precode</td>
```

Comment:
this is specifically for the `jdbc` interpreter? if so, suggest name `zeppelin.jdbc.precode` or `zeppelin.jdbc.sql.precode`